src/design-system/components/Prompt/Prompt.tsx

```diff
@@ -98,9 +102,10 @@ export const Prompt = ({
                   exit={{ opacity: 0 }}
                   transition={deceleratedShort}
                   background={background ?? 'surfaceMenu'}
-                  borderRadius="16px"
+                  borderRadius={borderRadius}
```

Comment:
Can you do `16px` as default if not provided ? I see we use `<Prompt />` in a lot of places.

![image](https://github.com/rainbow-me/browser-extension/assets/53529533/4e0a5f72-ac5d-4ce1-9331-c4b4cf134a22)